### DISCUSSION TITLE
Do you consider adding product rating to API for Catalog Items?

### DISCUSSION THREAD

**Discussion by MarekMalczewskiWavemaker** [amzn/selling-partner-api-models#622] [Ideas] [answered]
Hi Guys

I am currently trying to process product data, but unfortunately I can't see product ratings in "GET https://sellingpartnerapi-na.amazon.com/catalog/2020-12-01/items" endpoint results.

Do you consider adding it? Would be great :).

![Zrzut ekranu 2021-08-25 081812](https://user-images.githubusercontent.com/89507023/130736749-749b9653-0ce7-418d-880b-2d1b5fb30e21.jpg)

**Comment by krithic15** [comment, marked as answer]
Hello @MarekMalczewskiWavemaker,

Thank you for your suggestion. We have submitted your feature request to the appropriate team.
Thanks,
Krithi, Solutions Architect
Selling Partner Developer Services

**Comment by MarekMalczewskiWavemaker** [comment]
Fantastic!
Thank you.
